Product/Production/Services/DocumentRetrieveCore/src/test/java/gov/hhs/fha/nhinc/docretrieve/audit/transform/DocRetrieveAuditTransformsTest.java

```diff
@@ -0,0 +1,358 @@
+                equals(AuditTransformsConstants.ACTIVE_PARTICIPANT_ROLE_CODE_SOURCE_DISPLAY_NAME)) {
+                sourceActiveParticipant = item;
+            }
+        }
+
+        if (isRequesting) {
+            assertEquals(remoteObjectIP, sourceActiveParticipant.getUserID());
+        }
+
+        if (isRequesting) {
+            assertEquals(ManagementFactory.getRuntimeMXBean().getName(), sourceActiveParticipant.getAlternativeUserID());
+        }
+        assertEquals(isRequesting, sourceActiveParticipant.isUserIsRequestor());
+        if (isRequesting) {
+            assertEquals(localIP, sourceActiveParticipant.getNetworkAccessPointID());
+        } else {
+            assertEquals(webContextProperties.getProperty(NhincConstants.REMOTE_HOST_ADDRESS),
+                sourceActiveParticipant.getNetworkAccessPointID());
+        }
+        assertEquals(AuditTransformsConstants.NETWORK_ACCESSOR_PT_TYPE_CODE_NAME,
+            sourceActiveParticipant.getNetworkAccessPointTypeCode());
+        assertEquals(AuditTransformsConstants.ACTIVE_PARTICIPANT_ROLE_CODE_SOURCE,
+            sourceActiveParticipant.getRoleIDCode().get(0).getCode());
+        assertEquals(AuditTransformsConstants.ACTIVE_PARTICIPANT_CODE_SYSTEM_NAME,
+            sourceActiveParticipant.getRoleIDCode().get(0).getCodeSystemName());
+        assertEquals(AuditTransformsConstants.ACTIVE_PARTICIPANT_ROLE_CODE_SOURCE_DISPLAY_NAME,
+            sourceActiveParticipant.getRoleIDCode().get(0).getDisplayName());
+    }
+
+    /**
+     * This method is override in this test class in order to check RD specific testing
+     *
+     * @param request
+     * @param isRequesting
+     * @param localIP
+     * @param webContextProperties
+     * @param remoteObjectIP
+     */
+    protected void testGetActiveParticipantDestination(LogEventRequestType request, Boolean isRequesting, String localIP,
+        Properties webContextProperties, String remoteObjectIP) {
+        ActiveParticipant destinationActiveParticipant = null;
+        for (ActiveParticipant item : request.getAuditMessage().getActiveParticipant()) {
+            if (item.getRoleIDCode().get(0).getDisplayName() != null && item.getRoleIDCode().get(0).
+                getDisplayName().equals(AuditTransformsConstants.ACTIVE_PARTICIPANT_ROLE_CODE_DESTINATION_DISPLAY_NAME)) {
+                destinationActiveParticipant = item;
+            }
+        }
+        if (!isRequesting) {
+            assertEquals(AuditTransformsConstants.ACTIVE_PARTICIPANT_USER_ID_SOURCE, destinationActiveParticipant.getUserID());
+        } else {
+            assertEquals(webContextProperties.getProperty(NhincConstants.WEB_SERVICE_REQUEST_URL),
+                destinationActiveParticipant.getUserID());
+        }
+
+        assertEquals(!(isRequesting), destinationActiveParticipant.isUserIsRequestor());
+        assertEquals(localIP,
+            destinationActiveParticipant.getNetworkAccessPointID());
```

Comment:
Converse of previous wrapping comments -- don't wrap statements if they will fit on a single line.